locales-src/asrouter.ftl

```diff
@@ -90,17 +90,42 @@ cfr-whatsnew-button =
   .label = What’s New
   .tooltiptext = What’s New
 
+cfr-whatsnew-fx70-hero-title = { -brand-short-name } now fights harder for your privacy
+cfr-whatsnew-fx70-hero-body = The latest update enhances the Tracking Protection feature and makes it easier than ever to create secure passwords for every site.
+
+cfr-whatsnew-fx70-notes-body = Protection reports, Secure Proxy for public Wifi, improved account controls, and many more safety and security tools are included in the latest update.
+cfr-whatsnew-fx70-notes-link-text = Read the release notes
+
+cfr-whatsnew-tracking-protect-title = Protect yourself from trackers
+cfr-whatsnew-tracking-protect-body = { -brand-short-name } blocks many common social and cross-site trackers that follow what you do online.
+cfr-whatsnew-tracking-protect-link-text = View your protection report
+
 # This string is displayed before a large numeral that indicates the total number
 # of tracking elements blocked. Don’t add $blockedCount to
 # your localization, because it would result in the number showing twice.
-cfr-whatsnew-tracking-protections-title =
+cfr-whatsnew-tracking-blocked-title =
   { $blockedCount ->
     [one] Tracker blocked
    *[other] Trackers blocked
   }
-cfr-whatsnew-tracking-protections-subtitle = Since { DATETIME($earliestDate, month: "long", year: "numeric") }
+cfr-whatsnew-tracking-blocked-subtitle = Since { DATETIME($earliestDate, month: "long", year: "numeric") }
+cfr-whatsnew-tracking-blocked-link-text = View Protection Report
+
+cfr-whatsnew-lockwise-backup-title = Back up your passwords
+cfr-whatsnew-lockwise-backup-body = Now generate secure passwords you can access anywhere you sign in.
+cfr-whatsnew-lockwise-backup-link-text = Turn on backups
+
+cfr-whatsnew-lockwise-take-title = Take your passwords with you
+cfr-whatsnew-lockwise-take-body = The { -lockwise-brand-short-name } mobile app lets you securely access your backed up passwords from anywhere.
```

Comment:
Is this a correct usage of another message id as it's not quite simply a plain noun… ?

Reply:
This is not a normal message, it's a term, and it's designed to be only referenced in other messages.

Reply:
I was more concerned about it being used in the context of both "the" article and modifying "mobile app," but I suppose that's the point of fluent parameters to allow localizations to decide if there should be cases on the term ?

Reply:
Hardly any language can use that structure, so it will likely end up with something like "the mobile app (of) Lockwise)". Anyhow, that's expected and parameterized terms can help work around it.